src/game/objects/har.c

```diff
@@ -590,6 +590,24 @@ void har_move(object *obj) {
                     har_finished(obj);
                 }
             }
+        } else if(h->state != STATE_WALKFROM && h->state != STATE_WALKTO) {
+            // add some friction from the floor if we're not walking
+            // This is important to dampen/eliminate the velocity added from pushing away from the other HAR
+            if(vel.x > 0.0f) {
+                if(vel.x - 0.2f > 0.0f) {
```

Comment:
This could be an edge function btw ;)

Reply:
what is that?

Reply:
Sorry, step function. Brain fart. E.g. https://thebookofshaders.com/glossary/?search=step